fbw-a380x/src/systems/instruments/src/PFD/FMA.tsx

```diff
@@ -69,6 +72,38 @@ export class FMA extends DisplayComponent<{ bus: EventBus; isAttExcessive: Subsc
 
   private AB3Message = Subject.create(false);
 
+  private isUnrestrictedClimbDescent: number = 0;
```

Comment:
You don't need this property, you can just call `this.unrestrictedClimbDescent.get()`

Reply:
Ohh, smart, thanks!